storage-operator/deploy/role.yaml

```diff
@@ -59,3 +60,9 @@ rules:
   - '*'
   verbs:
   - '*'
+- apiGroups:
+  - storage.k8s.io
+  resources:
+  - storageclasses
+  verbs:
+  - '*'
```

Comment:
Only `GET`, no?

Reply:
I'll try, but yeah that should be enough.

Reply:
Not enough, need `get`, `list` and `watch`.

Done